src/tests-tokens/strict-mode-tests.ts

```diff
@@ -257,6 +262,60 @@ describe('token metadata strict mode', () => {
     expect(entry.result?.processed).toBe(false);
   });
 
+  test('metadata timeout errors get retried', async () => {
+    const mockTokenUri = {
+      okay: true,
+      result: cvToHex(stringCV('http://indigo.com/nft.jpeg', 'ascii')),
+    };
+    nock('http://127.0.0.1:20443')
+      .post(
+        '/v2/contracts/call-read/SP176ZMV706NZGDDX8VSQRGMB7QN33BBDVZ6BMNHD/project-indigo-act1/get-token-uri'
+      )
+      .reply(200, mockTokenUri);
+    nock('http://indigo.com')
+      .get('/nft.jpeg')
+      .delay(METADATA_FETCH_TIMEOUT_MS + 1000)
```

Comment:
Perhaps we make `METADATA_FETCH_TIMEOUT_MS` env var configurable so that we can set it to a low value here and avoid the test taking (10 + 1) seconds to complete